test/library/format_as_text.spec.jsx

```diff
@@ -1,51 +1,60 @@
 import React from 'react';
+import { render as rtlRender, screen, renderHook, fireEvent } from '@testing-library/react';
 
-import { mount } from '../test_util';
+import { mount, render } from '../test_util';
 import NumericFormat from '../../src/numeric_format';
 import PatternFormat from '../../src/pattern_format';
 
 /*** format_number input as text ****/
 describe('NumberFormat as text', () => {
   it('should format numbers to currency', () => {
-    const wrapper = mount(
+    rtlRender(
       <NumericFormat value={2456981} displayType={'text'} thousandSeparator={true} prefix={'$'} />,
     );
-    expect(wrapper.find('span').text()).toEqual('$2,456,981');
+
+    const span = screen.getByTestId('rnf-renderText-span');
```

Comment:
I don't think all the changes merged in #763 are ready to be merged into main yet. I think it would be better to close this PR and create a new one when some of the other ongoing work in the dev branch has been completed.